composer.json

```diff
@@ -95,13 +94,10 @@
         "pulse00/ffmpeg-bundle": "^0.6",
         "symfony/browser-kit": "^3.4",
         "symfony/css-selector": "^3.4",
-        "symfony/debug": "^3.4",
         "symfony/debug-bundle": "^3.4",
         "symfony/dom-crawler": "^3.4",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/monolog-bridge": "^3.4",
         "symfony/phpunit-bridge": "^3.4.8",
-        "symfony/process": "^3.4",
```

Comment:
This seems to be used in the [`bin/runtests` script](https://github.com/sulu/sulu/blob/develop/bin/runtests#L21), so it should probably stay.

Reply:
fixed